lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
I still think having errors in this section is a bad idea. Our output already shows the rule the message comes from, I don't think we really need to force people into a situation like this.